### ISSUE TITLE
Support for Android TV & Landscape Layout

### ISSUE THREAD

**Issue by peat80** [maxrave-dev/SimpMusic#164]
It would be awesome if the app could support androidtv in a future version.

I have installed it on my shieldtv and it almost works already.
The mini player is not really reachable without touch controls and there is of course no launcher shortcut but otherwise it works okay already. 

A landscape mode is missing so it looks a little weird.
Adding that would also help people that would like to use the app on a tablet.

**Comment by DarkCrypt** [comment]
I agree with this. I have the app installed on Android TV 10 and a lot of it works already.

**Comment by maxrave-dev** [comment]
I will check it

**Comment by RayzaEFC** [comment]
> I will check it

I second this, an Android TV version would be amazing. Please can you do this maxrave-dev

**Comment by maxrave-dev** [comment]
Need more contributor 🤧🤧🤧

**Comment by CrazyWolf13** [comment]
Duplicate of [https://github.com/maxrave-dev/SimpMusic/issues/731](https://github.com/maxrave-dev/SimpMusic/issues/731)
